app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
This one is good practice. Actually, if you use any of the rails generators they will add it automatically so if anything we need to change rubocop to enforce it. Most dfe projects already do this. Here is more info on what it does: https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do . Again, we need a new pr to add it to everything. I can take it off now so that we can add it to everything in a separate pr

Reply:
Alright, let's keep it.